packages/@ama-sdk/swagger-builder/README.md

```diff
@@ -106,6 +106,3 @@ A code generator is provided to generate an extension of the **Amadeus Digital E
 ```bash
 yarn yo @ama-sdk/sdk:api-extension
```

Comment:
same here, remove reference to Amadeus Digital (line 104) ?

`A code generator is provided to generate an API specification extension.`